### ISSUE TITLE
[IDEA] Advanced ALM

### ISSUE THREAD

**Issue by BenediktBergmann** [Power-Platform-DevOps/Power-Platform-ALM-Starter-Kit#7]
The ALM Starter Kit could describe possible additions to a basic ALM process to make it more advanced.

Those could for example be
- automated Tests
-- Unit
-- Integration
-- UI
- Checks against the Solution Checker

**Comment by rpothin** [comment]
I totally agree with this idea. But perhaps we can work on that point for a later version of the starter kit.

What is your point of view on that?
